package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "6.6.3",
+  "version": "6.7.4",
```

```suggestion
  "version": "6.7.0",
```

Per [semver](https://semver.org), shouldn't this be `6.7.0`?

---

@f1337 I based the patch increment on the PR in the Queue prior to this PR per this [Slack](https://dsva.slack.com/archives/CBU0KDSB1/p1597149227305700). I think their minor increment should be `6.7.0` making this patch increment `6.7.1`.   Getting the version of these correct with multiple PRs that might be merged at different times seems difficult.   Thanks!

---

Changed version to 6.7.1 based on 6.7.0 set in #472